js-src/aws/s3Upload.ts

```diff
@@ -19,20 +21,38 @@ export const uploadFiles = (files: FileList) => {
             }
         });
 
-        const bucket = "tdr-files";
+        const currentUser = getCurrentUser();
```

Comment:
We already looked up the current user in `getSession()` on line 13, so you could refactor the code to reuse the user, and avoid the placeholder username on line 26.

It's probably easier to show you what I mean - give me a shout if you want to pair on it.

Reply:
Refactored the code